setup.py

```diff
@@ -1,15 +1,420 @@
+import os
+import sys
+import subprocess
+import tarfile
+from shutil import copy
+from platform import python_version
+
+try:
+    # wget module is required to download SUNDIALS or SuiteSparse and
+    # is not a core requirement.
+    import wget
+
+    NO_WGET = False
+except (ImportError, ModuleNotFoundError):
+    NO_WGET = True
 try:
     from setuptools import setup, find_packages
 except ImportError:
     from distutils.core import setup, find_packages
+from distutils.cmd import Command
+
+
+def download_extract_library(url):
+    # Download and extract archive at url
+    if NO_WGET:
+        # The NO_WGET is set to true if the wget could not be
+        # imported.
+        error_msg = "Could not find wget module. Please install wget module."
```

Comment:
can you provide instructions on how to install in this error message? edit: I got this error even after installing wget

Reply:
Oh yes, the error message should be
`Could not find wget module. Please install wget module (pip install wget).`
This uses the `wget` python package (https://pypi.org/project/wget/). I did not add it to the list of pybamm dependencies because... well it's not really a dependency.
I hope that in a near future, I'll find the time to replace `wget` by the `urllib` module which is in the standard library.
Also I should update the installation notes, because I realize now `wget` is not mentioned anywhere